### PR TITLE
responsive layout

### PR DESCRIPTION
**Pull request by targonskii**
Added icons in sprite. 
Made responsive layout for header, main and footer.

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -5,50 +5,84 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
+    <!-- <link rel="icon" href="./images/svg-sprite.svg"> -->
     <title>What do we offer</title>
 </head>
 <body>
+    <div class="wrapper">
         <header>
             <div class="header__main">
```
Comment (ghost) [inline Triphouse-project/index.html:14]: 2021-04-19_21h53_42
Your header doesn't right centered, you have padding only left and top sides.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -5,50 +5,84 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (ghost) [inline Triphouse-project/index.html:5]: 2021-04-19_21h58_59
Please align content

File: Triphouse-project/index.html (hunk 1 above)
Comment (targonskii) [reply Triphouse-project/index.html:14]: Added right padding and center aligned header

File: Triphouse-project/index.html (hunk 2 above)
Comment (targonskii) [reply Triphouse-project/index.html:5]: Aligned content

**Pull request merged**